unit_test/test_spdm_crypt/test_spdm_crypt.c

```diff
@@ -35,6 +35,112 @@ uint8_t m_libspdm_subject_alt_name_buffer3[] = {
 uint8_t m_libspdm_dmtf_oid[] = { 0x2B, 0x06, 0x01, 0x4,  0x01,
                                  0x83, 0x1C, 0x82, 0x12, 0x01 };
 
+/* rsa2048/end_requester.cert.der
+ * https://lapo.it/asn1js/#MIIEDzCCAnegAwIBAgIBAzANBgkqhkiG9w0BAQsFADAtMSswKQYDVQQDDCJETVRGIGxpYnNwZG... */
+uint8_t m_cert_buffer[] = {
```

Comment:
please dont hardcode buffer. Please read file.

Reply:
Thank , I have modified it